packages/svelte/src/runtime/store/index.js

```diff
@@ -116,6 +122,10 @@ export function writable(value, start = noop) {
  */
 
 /**
+ * Derived value store by synchronizing one or more readable stores and
+ * applying an aggregation function over its input values.
+ *
+ * https://svelte.dev/docs/svelte-store#derived
```

Comment:
This is the implementation that AFAIK doesn't show up, so shouldn't have any docs
```suggestion
```